packages/relay/tests/lib/services/cacheService/cacheService.spec.ts

```diff
@@ -158,7 +159,40 @@ describe('CacheService Test Suite', async function () {
       entries['key2'] = 'value2';
       entries['key3'] = 'value3';
 
-      cacheService.multiSet(entries, callingMethod, undefined, false);
+      cacheService.multiSet(entries, callingMethod, undefined, undefined, false);
+      mock
+        .stub(cacheService, 'getAsync')
+        .onFirstCall()
+        .returns(entries['key1'])
+        .onSecondCall()
+        .returns(entries['key2'])
+        .onThirdCall()
+        .returns(entries['key3']);
+
+      for (const [key, value] of Object.entries(entries)) {
+        const valueFromCache = cacheService.getSharedWithFallback(key, callingMethod, undefined);
+        expect(valueFromCache).eq(value);
+      }
+    });
+
+    it('should be able to set using pipelineSet and get them separately using internal cache', async function () {
+      process.env.MULTI_SET = 'false';
```

Comment:
The only difference between the two tests is the value of `MULTI_SET`
How can we confirm whether pipeline was used or not?

Reply:
Only thing I can think of, if we expose getter for this flag -> `MULTI_SET` in the cache service. And then call and check this flag, depending on the case. If it's false, it will always use pipeline.

Reply:
You could confirm via log mock.
Example from Logans recent PR https://github.com/hashgraph/hedera-json-rpc-relay/pull/2243/files#diff-62cee0fdf114bfe63346e4562cdf07ee2cec616ad8ba07784347d8b49fa09d2fR226

Reply:
Thanks, opened issue to track it. https://github.com/hashgraph/hedera-json-rpc-relay/issues/2267